packages/ngx-calendar/src/lib/components/calendar/calendar.component.ts

```diff
@@ -88,7 +88,14 @@ export class CalendarComponent implements OnInit, OnChanges {
       this.activeDate = this.selectedDate;
       this.updateHeaderLabel();
     } else {
-      this.activeDate = DateHelper.parseDate(new Date());
+      let activeDateToSet = new Date();
+
+      if (this.interval && this.interval.isInRange(activeDateToSet)) {
```

Comment:
@jsebrech, at first I thought it was !this.interval.isInRange but when I looked at the date.interval.ts implementation 
`isInRange(value: Date): boolean {
    return this.meaning === 'OR' ? this.checkIsInRange(value) : !this.checkIsInRange(value);
  }` it is uses the falsey expression in case of a 'NOR'